src/schema/notifications.ts

```diff
@@ -127,7 +153,9 @@ export class ZimbraNotifications {
 					}
 				});
 				searchResponse[query].contacts = uniqBy(
-					[item].concat(searchResponse[query].contacts),
+					[item]
+						.concat(searchResponse[query].contacts)
+						.sort((a, b) => doSort(a, b, sortByValue)),
```

Comment:
It's better to insert the item instead of sorting every time when a new contact is added because, `searchResponse[query].contacts` is previous cached data which we can believe already sorted